examples/pyportal_simpletest.py

```diff
@@ -14,7 +14,7 @@
 pyportal = PyPortal(url=TEXT_URL, status_neopixel=board.NEOPIXEL)
 
 # Set display to show REPL
-board.DISPLAY.show(None)
+board.DISPLAY.root_group = None
```

Comment:
At one point, I believe a .show(None) displayed the REPL but I don't think it does anymore. I believe setting the root_group to None will display a blank screen. In order to display the REPL I think you need to use CIRCUITPYTHON_TERMINAL.